### PR TITLE
[DIA-1523] Replace loadurl with evaluatejavascript

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

**Comment by carmelo-iriti** [comment]
@Nevazhnovu before merging, make sure the tests are passing

**Review by carmelo-iriti — APPROVED** [review]

**Comment by carmelo-iriti** [comment]
@Nevazhnovu I just merged new commits on develop, please **rebase** (instead of **merge**) your mr on develop

**Comment by andresilveirah** [comment]
@carmelo-iriti I rebased the branch. Is there anything else missing on this PR?

**Comment by carmelo-iriti** [comment]
@andresilveirah  everything it’s there, the mr is complete

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt [hunk 1]
```diff
@@ -67,7 +67,7 @@ internal class ConsentWebView(
     private fun setup() {
         when {
             viewId != null -> { id = viewId }
-            Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 -> { id = View.generateViewId() }
+            viewId == null -> { id = View.generateViewId() }
```
Comment (carmelo-iriti) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt:70]: Just for an extra layer of security, test manually if this point is reached when the id is set from the client app, currently we don’t have an integration test for it

**Review by carmelo-iriti — APPROVED** [review]

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt (hunk 1 above)
Comment (Nevazhnovu) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt:70]: Ok, one moment, will do

**Review by carmelo-iriti — APPROVED** [review]

**Pull request merged**